### DISCUSSION TITLE
We need volunteers for testing the Backend/Frontend

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#3816] [General]
I have identified several errors in the Backend and reported them, but I am not sure they are all. I would like to ask who can and has the time to go through the entire Backend and evaluate each section carefully. This action must be done as soon as possible.

I used it in my testing environment

- OpenMage 20.3.0
- Magento Sample Pack
- PHP 8.3
- MAGE_IS_DEVELOPER_MODE=1
- Log files system.log and exception.log activated in Backend

**Comment by ma4nn** [comment]
@ADDISON74 you mean to test OpenMage for PHP 8.3 compatibility?

**Reply by addison74** [reply]
Indeed for PHP 8.3. OpenMage has no major problems in my opinion, but there are places in the code where the deprecated ones must be fixed. I found some and I think there are others, but I am only one person in this investigation. A little help from others would be really helpful.
